app/lib/l10n/app_en.arb

```diff
@@ -63,14 +51,14 @@
   "onboarding_2_text": "Your body reacts to medications uniquely.\n\nMedications that are effective for a majority of people can be toxic to you.",
   "onboarding_3_header": "Genome power unlocked to improve human health",
   "onboarding_3_text": "PharMe warns about unintended drug responses according to your genome.\nThis enables you to avoid medications that are ineffective or have side-effects.",
-  "onboarding_3_disclaimer": "It does not recommend medications for specific symptoms or illnesses!",
+  "onboarding_3_disclaimer": "We DO NOT recommend medications for specific symptoms or illnesses!",
```

Comment:
Again the all-caps thing but I guess at least you are being consistent. At this point no strong opinion I guess, so will leave this up to you.